sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -348,6 +348,12 @@ export class ServiceBusClient {
   }
 }
 
+/**
+ * @internal
+ * @ignore
+ * @param {*} mode
+ * @returns {(mode is "peekLock" | "receiveAndDelete")}
+ */
 function isReceiveMode(mode: any): mode is "peekLock" | "receiveAndDelete" {
```

Comment:
Oh wow, I'm surprised even things not marked as exported are getting documented. I wonder if the version of TypeDoc being used was updated...

Reply:
@KarishmaGhiya